package.json

```diff
@@ -17,11 +17,11 @@
   "author": "Blackbaud",
   "license": "ISC",
   "dependencies": {
-    "@blackbaud/help-client": "1.1.0"
+    "@blackbaud/help-client": "1.2.0"
   },
   "devDependencies": {
-    "@blackbaud/skyux-builder": "^1.0.2",
-    "@blackbaud/skyux": "^2.0.0"
+    "@blackbaud/skyux-builder": "1.10.0",
+    "@blackbaud/skyux": "2.10.0"
```

Comment:
How closely do you want the versions to be to "latest"? Ux is at 2.11 and Builder's at 1.13. Just curious...